File: Pset01/mario2.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int x;
    do{
        printf("Height: ");
        x = get_int();
    }
    while(x>23 || x<0);

    for (int i=0 ; i < x ; i++)
    {
        for(int y = 0 ; y <x-i-1 ; y++)
            printf(" ");
        for(int y = 0 ; y <= i ; y++)
            printf("#");
        printf("  ");
        for(int y = 0 ; y <= i ; y++)
            printf("#");
        printf("\n");
    }
}

File: pset06/mario2.py
import cs50

print ("Height: ",end="")
height = cs50.get_int()
while height < 0 or height >23:
    print ("Height: ",end="")
    height = cs50.get_int()
for lines in range (2,height+2):
    for spaces in range (height+1-lines):
        print(" ",end="")
    for hashes in range (lines):
        print("#",end="")
    print("")